File: Programmers/LV1/정리 필요/핸드폰번호가리기.py
def solution(phone_number):
    answer = ''

    length = len(phone_number)

    # print(phone_number[0:(len(phone_number)-4)])

    # a = phone_number[0:(len(phone_number) - 4)]

    answer = '*' * (length -4 )

    answer += phone_number[-4:]

    # b = phone_number[len(phone_number)-4 : len(phone_number)]

    # print(a)
    # print(b)

    return answer

print(solution("0277788888"))

File: Programmers/LV1/JAVA/핸드폰_번호_가리기.java
package Programmers.level1;

import java.util.ArrayList;

public class 핸드폰_번호_가리기 {

	public static void main(String[] args) {

		String answer = "";
		String phone_number = "01033334444";
		String p = "027778888";
		int number_length = phone_number.length();
		int plen = p.length();
		
		for(int i=0; i<number_length-4; i++) {
			answer+="*";
		}
		
		answer += phone_number.substring(number_length-4);
		
		System.out.println(answer);
		
		

	}

}
